src/pages/Home/Home.js

```diff
@@ -0,0 +1,90 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import {Link, NavLink} from "react-router-dom";
+import Redditlogo from "../../assets/logo.png"
+
+function Home(props) {
+const [endpoint,setEndpoint] = useState(`https://www.reddit.com/hot.json?limit=15`)
+const [posts,setPosts]=useState([])
+
+
+
+
+useEffect(()=>{
+    async function fetchData(){
+        try {
+            const result = await axios.get(endpoint)
+            setPosts(result.data.data.children)
+            console.log(result)
+        } catch (e) {
+            console.error(e)
+        }
+    }
+    fetchData()
+    },[])
+
+
+
+    return (
+        <>
+            <body className="outer-container">
+            <header className="outer-container header">
+            <div className="inner-container ">
+                <nav className="inner-container navbar">
+                    <ul className="nav-list">
+                        <li className="nav-link"><NavLink to={"/"}>Hottest Posts</NavLink></li>
+                        <a href="https://www.reddit.com"><li>Reddit</li></a>
+                        <li><NavLink to={"/Subreddit/r/memes"}>Memes</NavLink></li>
+                    </ul>
+                </nav>
+
+            <div className="inner-container logo-title">
+                <img className="reddit-logo" src={Redditlogo} alt="logo image"/>
+            </div>
+                <h1 className="reddit-title">Reddit</h1>
+
+            </div>
+            </header>
+            {Object.keys(posts).length > 0 &&
```

Comment:
Deze check kun je dan wat simpeler maken
```suggestion
            {posts &&
```